src/Compute/Compute/Generated/Models/PSGalleryImageVersion.cs

```diff
@@ -40,7 +40,8 @@ public string ResourceGroupName
             }
         }
 
-        public GalleryImageVersionPublishingProfile PublishingProfile { get; set; }
+        // This is commented out for hiding the breaking change caused by the client library update.
+        //public GalleryImageVersionPublishingProfile PublishingProfile { get; set; }
```

Comment:
@hyonholee what's the breaking change that we're trying to hide here? Since we're commenting this property out, we're introducing a breaking change. Since this is a preview module, we can just make the breaking change introduced by the SDK.

Reply:
GalleryImageVersionPublishingProfile used to have "source" property in 2019-03-01 API version.

https://github.com/Azure/azure-rest-api-specs/blob/d8e61eccaaa0f9ddeb95dfed0a6dcebe06dbbc3b/specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/gallery.json#L1438

Now, galley team moved the property to GalleryImageVersionStorageProfile in 2019-07-01 version.

https://github.com/Azure/azure-rest-api-specs/blob/d8e61eccaaa0f9ddeb95dfed0a6dcebe06dbbc3b/specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/gallery.json#L1154